services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -361,6 +361,73 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with Matchers with Mocki
       }
     }
 
+    "getFiles" should {
+      val dummyFilesTree: List[FileTree] = List(dummyFileTree)
+      val path = Paths.get(dummyFilesTree.head.path)
+      val encodedPathStr = URLEncoderUtils.encode(path.toString)
+      //        val emptyListValidateResponse = "List((,List(JsonValidationError(List(error.expected.jsarray),WrappedArray())))"
```

Comment:
Why do we need it? Never commit an unused commented code

Reply:
Yep, my fault. Fixed.